### PR TITLE
Additional info from Keras.io

### PR DESCRIPTION
**Pull request by elainecrutchley**

### CODE REVIEW

**Comment by rthadur** [comment]
@elainecrutchley can you please sign CLA

**Comment by gbaned** [comment]
@elainecrutchley gentle ping to sign cla

**Comment by elainecrutchley** [comment]
can I have a link to the document?

**Review by tanzhenyu — COMMENTED** [review]
Why do we need this in __init__ file?

**Comment by gbaned** [comment]
@elainecrutchley Did you get a chance to look on reviewer comments? Please let us know on the update. Thanks!

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this PR still valid? Assigning the `stalled` label. Please comment to reassure me that this is still being worked on.

**Comment by gbaned** [comment]
I'm going to go ahead and close this PR, because it seems to have stalled. If you're still interested in pursing this (and responding to my comments), please feel free to reopen!
